utils.go

```diff
@@ -1,12 +1,41 @@
 package us3
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	uerr "github.com/ucloud/ucloud-sdk-go/ucloud/error"
+	us3 "github.com/ufilesdk-dev/ufile-gosdk"
```

Comment:
Ops, so confusing name of `ufile` here. Strange sdk.

Reply:
The predecessor of us3 is ufile.